backend/src/modules/tournament-tree/core/domain/TournamentTree.ts

```diff
@@ -72,8 +74,16 @@ export class TournamentTree {
     return [...this.tournamentTreeArray];
   }
 
-  public getMatchesQueueReadyToBegin(): FightingTeamsGroup[] {
-    return this.tournamentTreeArray.filter(({ firstTeam, secondTeam }) => firstTeam && secondTeam);
+  public getMatchesQueueReadyToStart(): MatchReadyToStart[] {
+    return this.tournamentTreeArray
+      .filter(({ firstTeam, secondTeam }) => firstTeam && secondTeam)
+      .map((match) => {
+        return new MatchReadyToStart({
+          firstTeam: match.firstTeam as TournamentTeam,
+          secondTeam: match.secondTeam as TournamentTeam,
+          matchNumber: match.matchNumberInSequence as number,
+        });
+      });
```

Comment:
Fajnie, - dzięki o wytłumaczenie, że chodzi o to udefined!
Fajnie, że to tak rozłożyłeś.
Testy w całej aplikacji przechodzą?